common/model/query/query/defaults.js

```diff
@@ -1,10 +1,12 @@
+import { Sort } from '../sort'
+
 import { displayTypes, keys, modes } from './keys'
 
 export const defaults = {
   [keys.mode]: modes.raw,
   [keys.displayType]: displayTypes.table,
   [keys.filter]: null,
-  [keys.sort]: [],
+  [keys.sort]: Sort.create(),
```

Comment:
what is the motivation to use a function?

Reply:
to do not expose the model state